samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/services/assessment/AssessmentService.java

```diff
@@ -1118,6 +1118,191 @@ public SecurityAdvice isAllowed(String arg0, String arg1,
 			return text;
 		}
 
+
+	/**
+	 * 
+	 * @param assessment
+	 * @param bundle
+	 * @return
+	 */
+	public String exportAssessmentToMarkupText(AssessmentFacade assessment, Map<String,String> bundle) {
+		StringBuffer markupText = new StringBuffer(); 
+		int nQuestion = 1;
+		
+		Iterator<SectionFacade> sectionIter = assessment.getSectionArray().iterator();
```

Comment:
`Iterator<>` not necessary here, use a `for` loop to iterate over the items in the `Set<>` or `List<>`